otwcode/otwarchive
# AO3-5884 Restrict ability to search for and update users

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-5884

## Purpose

What does this PR do?

[Diff for relevant commits](https://github.com/otwcode/otwarchive/pull/3838/files/86a3dfbf4185073e6d119b9b5390596225ddb413..e38220c356a6473a6f5e958f69180b9c99e058d0) for ease of reviewing

## Testing Instructions

Sign in as an admin with no roles, and visit the pages referenced in this ticket-- https://otwarchive.atlassian.net/browse/AO3-5884 -- and ensure that admin cannot access the pages they are not supposed to.

Sign in as an admin with the superadmin or policy_and_abuse role, and ensure you can access these pages and the appropriate functions work for this admin.


## References

Are there other relevant issues/pull requests/mailing list discussions?

## Credit

*What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?*

(If you have a Jira account, please include the same name in the "Full name"
field on your Jira profile, so we can assign you the issues you're working on.)

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*

(If you do not fill in this section, we will use your GitHub account name and
they/them pronouns to avoid making assumptions about your gender.)
